cmd/server/pactasrv/initiative_user_relationship.go

```diff
@@ -0,0 +1,75 @@
+package pactasrv
+
+import (
+	"context"
+
+	"github.com/RMI/pacta/cmd/server/pactasrv/conv"
+	"github.com/RMI/pacta/db"
+	"github.com/RMI/pacta/oapierr"
+	api "github.com/RMI/pacta/openapi/pacta"
+	"github.com/RMI/pacta/pacta"
+	"go.uber.org/zap"
+)
+
+// Returns all initiative user relationships for the user that the user has access to view
+// (GET /initiative/{id}/user-relationships)
+func (s *Server) ListInitiativeUserRelationshipsByUser(ctx context.Context, request api.ListInitiativeUserRelationshipsByUserRequestObject) (api.ListInitiativeUserRelationshipsByUserResponseObject, error) {
+	// TODO(#12) Implement Authorization
+	iurs, err := s.DB.InitiativeUserRelationshipsByUser(s.DB.NoTxn(ctx), pacta.UserID(request.UserId))
+	if err != nil {
+		return nil, oapierr.Internal("failed to retrieve initiative user relationships by user", zap.Error(err))
+	}
+	result, err := dereference(mapAll(iurs, conv.InitiativeUserRelationshipToOAPI))
+	if err != nil {
+		return nil, err
+	}
+	return api.ListInitiativeUserRelationshipsByUser200JSONResponse(result), nil
+}
+
+// Returns all initiative user relationships for the initiative that the user has access to view
+// (GET /initiative/user-relationships/{id})
```

Comment:
This URL scheme is not intuitive to me, e.g. the difference between:

- `GET /initiative/{id}/user-relationships`, and
- `GET /initiative/user-relationships/{id}`, and

The benefit of RESTful stuff is that usually it's pretty obvious from the URL what the endpoint does, but I'm not getting that from these. How about something like:

- `GET /initiative/{id}` - Gets the initiative and the relationships, can optionally hide the latter behind a query param like `withRelationships=true`
	- If for some reason we're not frequently loading initiatives + their relationships at the same time, could also do `/initiative/{id}/relationships`, basically like it is now.
- `GET /user/{id}/initiatives` - Gets a list of initiatives the user has access to

Reply:
> If for some reason we're not frequently loading initiatives + their relationships at the same time, could also do /initiative/{id}/relationships, basically like it is now.

Yep! Public access won't load relationships.

I think this actually fits nicely into the "composite key exception" listed [here](https://github.com/stickfigure/blog/wiki/How-to-%28and-how-not-to%29-design-REST-APIs#rule-2-dont-add-unnecessary-path-segments)